ARE/services/WebService/src/main/java/eu/asterics/mw/webservice/RequestFilter.java

```diff
@@ -56,7 +56,7 @@
     private List<String> allowedOrigins = null;
 
     static {
-        AREProperties.instance.setDefaultPropertyValue(ARE_REST_ALLOWED_ORIGINS, "localhost,127.0.0.1,asterics.github.io,asterics-foundation.org", "Origins that are allowed to access the ARE REST API. Separate with comma (',').");
+        AREProperties.instance.setDefaultPropertyValue(ARE_REST_ALLOWED_ORIGINS, "localhost,127.0.0.1,asterics.github.io,asterics-foundation.org,asterics.eu,asterics.org,studyathome.technikum-wien.at", "Origins that are allowed to access the ARE REST API. Separate with comma (',').");
```

Comment:
@klues @sabicalija ok? Also added studyathome.technikum-wien.at to have it for testing purposes.